pkgs/by-name/de/devdash/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib
+, buildGoModule
+, fetchFromGitHub
+, nix-update-script
+}:
+
+buildGoModule rec {
+  pname = "devdash";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "Phantas0s";
+    repo = "devdash";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-RUPpgMVl9Cm8uhztdfKnuQ6KdMn9m9PewlT59NnTSiY=";
+  };
+
+  vendorHash = "sha256-xuc8rAkyCInNFxs5itwabqBe4CPg/sAuhcTJsapx7Q8=";
+
+  ldFlags = [
```

```suggestion
  ldflags = [
```

I can't seem to find a single definition using the camelCased version. The manual documents [this](https://nixos.org/manual/nixpkgs/stable/#var-go-ldflags) however. Is it even needed?

---

This lead to #262390

---

Right, seems like a typo on my end. I think I actually might be missing some version flags, so I'll add them and mark as resolved